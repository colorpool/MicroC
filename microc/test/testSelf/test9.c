void main(int n){
    char s;
    print 'c' 'a';
}