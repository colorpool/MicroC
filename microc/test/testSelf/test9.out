24 19 1 5 25 15 1 27 97 23 21 2